src/screens/nano-contract/NanoContractExecuteMethod.js

```diff
@@ -140,8 +144,8 @@ function NanoContractExecuteMethod() {
       if (typeToCheck !== 'Address') {
         continue;
       }
-      const addressInputValue = formRefs[i].current.value;
-      validateAddress(addressInputValue, formRefs[i].current);
+      const addressInputValue = formRefs[i].getValue();
+      validateAddress(addressInputValue, formRefs[i].ref);
```

Comment:
Shouldn't this be `formRefs[i].ref.current`? Is it still working when validating the address?

Reply:
You're right, that was my mistake. Fixed in https://github.com/HathorNetwork/hathor-wallet/pull/699/commits/37e16014eefa804dc9a627366f7ee2d7dca3b186. Note: this is only for custom address inputs, so it didn't fail for neither the sign address nor withdraw address inputs, only for Dozer's custom address arg. We should probably include those in the QA.